configure.ac

```diff
@@ -233,6 +234,9 @@ AS_IF([test "x$enable_require_system_font_provider" != xno  dnl
 
 
 # Locate and configure Assembler appropriately
+X86=false
+X86_64=false
+AARCH64=false
```

Comment:
Rather than tracking each CPU Arch in its own boolean variable, imho it would be better to replace them with a unified `cpu_family` variable and replace the `    AS_IF([test "x$X86" = xtrue], [...])     AS_IF([test "x$AARCH64" = xtrue], [...])` with an `AS_CASE([$cpu_family], ...)`.

Reply:
Adopted your (slightly modified) patchset.